homeassistant/components/vicare/sensor.py

```diff
@@ -177,13 +188,76 @@
         CONF_GETTER: lambda api: api.getCompressorHours(),
         CONF_DEVICE_CLASS: None,
     },
+    SENSOR_COMPRESSOR_HOURS_LOADCLASS1: {
+        CONF_NAME: "Compressor Hours Load Class 1",
+        CONF_ICON: "mdi:counter",
+        CONF_UNIT_OF_MEASUREMENT: None,
```

Comment:
`homeassistant.core.TIME_HOURS`, the same for the following `SENSOR_COMPRESSOR_HOURS_...`.

Reply:
Fixed